darkcluster/src/test/java/com/linkedin/darkcluster/TestDarkClusterManager.java

```diff
@@ -16,8 +16,7 @@
 import com.linkedin.r2.message.rest.RestRequestBuilder;
 
 import static com.linkedin.darkcluster.DarkClusterTestUtil.createRelativeTrafficMultiplierConfig;
-import static com.linkedin.darkcluster.TestDarkClusterStrategyFactory.DARK_CLUSTER_NAME;
-import static com.linkedin.darkcluster.TestDarkClusterStrategyFactory.SOURCE_CLUSTER_NAME;
+import static com.linkedin.darkcluster.TestDarkClusterStrategyFactory.*;
```

Comment:
nit: I think we want to avoid importing with *, right?

Reply:
Fixed. Thanks!